actors/user/src/main/java/org/sunbird/user/actors/UserMergeActor.java

```diff
@@ -161,25 +154,53 @@ private String getCustodianValue() {
   /**
    * This method creates Kafka topic for user-cert
    *
-   * @param requestMap
+   * @param mergeeId
+   * @param mergerId
    * @throws IOException
    */
-  private void updateUserCertDetails(Map<String, Object> requestMap) throws IOException {
+  private void mergeCertCourseDetails(String mergeeId, String mergerId) throws IOException {
     String content = null;
-    Map<String, Object> userCertMergeRequest = new HashMap<>();
-    userCertMergeRequest.put("messageType", "userCertMerge");
-    userCertMergeRequest.put("messageDetails", requestMap);
+    Telemetry userCertMergeRequest = createMergeAccountTask(mergeeId, mergerId);
```

Comment:
createMergeAccountTask to createAccountMergeTopicData

Reply:
Done